README.md

```diff
@@ -273,10 +273,6 @@ lua <<EOF
 require'nvim-treesitter.configs'.setup {
   highlight = {
     enable = true,
-    custom_captures = {
-      -- Highlight the @foo.bar capture group with the "Identifier" highlight group.
-      ["foo.bar"] = "Identifier",
-    },
```

Comment:
How does this change anything in the user experience? Wouldn't it by better to leave it in the config (since it uses the value at the right moment)? Especially since I think that changing the `hl_map` to late often does not have the desired outcome. We also remove that setting and tell users to use `hl_map` directly

Reply:
well this was a bit what i thought. I don't really like the map being set during the highlighter initialization (it might run a 1000 times during a user session which is not really expected).
I just put back the default custom_captures = {} in the config because it broke telescope. 
The reason why is to avoid plugins doing funky stuff like this:
https://github.com/nvim-telescope/telescope.nvim/blob/f262e7d56d37625613c5de0df5a933cccacf13c5/lua/telescope/previewers/utils.lua#L97-L116

Reply:
and yes i guess telling user to configure their hl_map directly would make more sense. but it's not documented in the neovim side yet.